Makefile

```diff
@@ -13,6 +13,7 @@ clean:
 	go clean -i ./...
 
 deps:
+	go get -u github.com/codegangsta/cli/...
 	go get -t ./...
```

Comment:
`go get -u github.com/codegangsta/cli/...` is not required. You should be building with `GOVENDOREXPERIMENT=1` which uses the `/vendor` directory for dependencies